.github/workflows/release.yml

```diff
@@ -102,6 +102,9 @@ jobs:
       - name: Update .zenodo.json url
         run: echo "`jq '.related_identifiers[0].identifier="https://github.com/ieeh-tu-dresden/powerfactory-tools/tree/${{ steps.cz.outputs.version }}"' .zenodo.json`" > .zenodo.json
 
+      - name: Update .CITATION.cff version number
```

Comment:
This can be done automatically by adding `".CITATION.cff:cff-version"` to `pyproject.toml`